mmv1/templates/terraform/custom_update/storage_folder_update.go.tmpl

```diff
@@ -0,0 +1,10 @@
+_ = config
+// we can only get here if force_destroy was updated
+if d.Get("force_destroy") != nil {
+    if err := d.Set("force_destroy", d.Get("force_destroy")); err != nil {
+        return fmt.Errorf("Error updating force_destroy: %s", err)
+    }
+}
```

Comment:
NIt: Since you're already adding this, you may want to go ahead and add the function that checks if this is the only field being changed (in case other fields are added later).

Reply:
for now this should be fine